### DISCUSSION TITLE
3 questions about upscale in Fooocus

### DISCUSSION THREAD

**Discussion by 925-Studio** [lllyasviel/Fooocus#1011] [Q&A] [answered]
1. How do I select ESRGAN model for different upscale purpose or just simply can't?
2. Can I change Denoise value when I try to upscale?
3. Drag image from above doesn't work (like the document showed), drag image from local drive works. Any ideas? (Chrome on MacOS)

Thanks in advance.

Additional quesiont: Can I customize the outputs folder to somewhere else?

**Comment by lllyasviel** [comment, marked as answer]
- Can I change Denoise value when I try to upscale?
if you know what you are doing, go to `dev tool` -> `force overwrite of denoising strength of upscale`
(but many people do not know what they are doing, and their knowledges learned from SD1.5 just does not work in SDXL upscale)
(SDXL latent is 1.384x smaller range and 2x larger, which means SDXL's denoise strength 0.25 will behave similar to strength 0.05 in SD1.5, SDXL's denoise strength 0.5 will behave similar to strength 0.2 in SD1.5)
(so, consider twice if you really know what you are doing)

- Drag image from above doesn't work
Mac support is experimental

- Can I customize the outputs folder to somewhere else?
Yes, see readme's `customization`

- How do I select ESRGAN model for different upscale purpose or just simply can't?
You cannot. But please share results if you can get results from any ESRGAN model better than Fooocus upscale model. But we tested lots of images and that never happens

**Reply by 925-Studio** [reply]
Wow, this is much more than I expected. Thanks a lot for your help, RESPECT!

**Reply by Apestein** [reply]
> * Can I change Denoise value when I try to upscale?
>   if you know what you are doing, go to `dev tool` -> `force overwrite of denoising strength of upscale`
>   (but many people do not know what they are doing, and their knowledges learned from SD1.5 just does not work in SDXL upscale)
>   (SDXL latent is 1.384x smaller range and 2x larger, which means SDXL's denoise strength 0.25 will behave similar to strength 0.05 in SD1.5, SDXL's denoise strength 0.5 will behave similar to strength 0.2 in SD1.5)
>   (so, consider twice if you really know what you are doing)
> * Drag image from above doesn't work
>   Mac support is experimental
> * Can I customize the outputs folder to somewhere else?
>   Yes, see readme's `customization`
> * How do I select ESRGAN model for different upscale purpose or just simply can't?
>   You cannot. But please share results if you can get results from any ESRGAN model better than Fooocus upscale model. But we tested lots of images and that never happens

What exactly is fooocus upscale model? Does it have a name? Where can I download it to use in comfyui for example? And is it still the best upscaler in 2025?
